File: src/syntax/lesson14_scanner/ScannerDemo.java
package syntax.lesson14_scanner;

import java.util.Scanner;

public class ScannerDemo {
    public static void main(String[] args) {

        System.out.print("Enter your name: ");
        Scanner scanner = new Scanner(System.in);
        String name = scanner.nextLine();

        System.out.print("Enter your age: ");
        int age = scanner.nextInt();

        System.out.println("My name is: " + name + "\n"
                + "I'm" + age + "years old.");


    }

}

File: src/syntax/lesson14_scanner/ScannerDemo.kt
package syntax.lesson14_scanner

import java.util.Scanner

fun main() {

    val scanner: Scanner = Scanner(System.`in`);


    print("Enter your name: ")
    val name = scanner.nextLine()
    println("Your name is $name")

    print("Enter your age: ")
    val age = scanner.nextInt()
    println("You are $age old")

    print("Enter your name by kotlin method: ")
    val nameByKotlin = readLine()
    println("Name by kotlin method: $nameByKotlin")

    print("Enter your age by kotlin method: ")
    val ageByKotlin = readln().toIntOrNull() ?: 0
    println("Age by kotlin method: $ageByKotlin")





}